versioned_docs/version-0.10/enterprise/install/releases.mdx

```diff
@@ -5,6 +5,30 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.12.0
+DATE TBD
+
+### Improvements
+- Use `ingressClassName` field for ingresses instead of the deprecated `kubernetes.io/ingress.class` annotation
+- Okteto daemonset no longer mounts host volumes nor runs in privilege mode
+- Improve pipeline jobs lifecycle and properly handle timeouts. Logs for pipelines that time out are no longer lost
+- Invalid ingresses not accepted by the nginx ingress controller are now caught by the webhook
+- Ingresses are put to sleep alongside all other namespace resources and automatically wake by with incoming requests
+- Upgrade to Okteto CLI 2.2.1
+- Upgrade ingress-nginx helm chart to 4.1.0
+- Deploying daemonset is no longer allowed by default
+- Account for "ready" container condition to check if a component is healthy
+- Add network quotas for okteto up
```

Comment:
can you link to the doc?

Reply:
@maroshii When we put the RC on teams we should merge the outstanding docs for this and then update the release notes PR, https://github.com/okteto/docs/pull/73/files